plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/MachineLocalizationConstant.java

```diff
@@ -26,9 +26,15 @@
     @Key("button.ok")
     String okButton();
 
+    @Key("button.save")
+    String saveButton();
+
     @Key("button.cancel")
     String cancelButton();
 
+    @Key("button.close")
+    String closeButton();
+
```

Comment:
all this button names can be used from org.eclipse.che.ide.CoreLocalizationConstant
no need to duplicate these